### DISCUSSION TITLE
Mobx form textfield default value not recognized on submit - Reactjs

### DISCUSSION THREAD

**Discussion by prajeeshbs** [mobxjs/mobx#3118] [Q&A]
The code is as follows

` <TextField {...form.$("phone.countryCode").bind()} value={222} />`
The form setup is as follows

```
{
        name: "phone",
        fields: [
          {
            name: "countryCode",
            label: "Country Code:",
            bindings: "TextField",
            placeholder: "+",
            rules: "required",
          }
       ]
}
```
The value displays correctly in the textfield. But when i submit the form, it shows "required" error.

Is assume field.input.onChange is not triggered when we set the value like this and hence the mobx field is not updated with the value.

Any idea on how to fix this?

**Comment by kubk** [comment]
It looks like the issue should be addressed to the form library you are using.
